packages/grafana-ui/src/components/Cascader/Cascader.test.tsx

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import { Cascader } from './Cascader';
 import { shallow } from 'enzyme';
```

Comment:
Could we also replace the exiting Enzyme tests with RTL so we don't have multiple frameworks in one test?

Reply:
I've removed the existing Enzyme test and replaced it with an RTL test that tests the search functionality.
It's not the same as the Enzyme test because the previous test was specifically testing the internal state (which is not what RTL's designed for) so I've tried to add in something useful for its replacement 😄